denguefever_tw/dengue_linebot/denguebot_fsm/denguebotfsm.py

```diff
@@ -235,7 +239,7 @@ def _create_zapper_imgmap(self, event):
             base_size=BaseSize(height=1040, width=1040),
             actions=[
                 MessageImagemapAction(
-                    text='我要綁定補蚊燈！',
+                    text='我要綁定捕蚊燈！',
```

Comment:
I prefer using `self.render_text` in this case os that we can configure it in `jinja2` files.
What your opinion on it?

Reply:
I see. It's better.